api/src/main/java/org/openmrs/module/patientflag_rest/patientflag_restActivator.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.patientflag_rest;
+
+
+import org.apache.commons.logging.Log; 
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.module.Activator;
+
+/**
+ * This class contains the logic that is run every time this module is either started or stopped.
+ */
+public class patientflag_restActivator implements Activator {
```

Comment:
Can we delete this? I think we already have a [module activator](https://github.com/openmrs/openmrs-module-patientflags/blob/master/api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java).